src/views/Gallery.vue

```diff
@@ -35,7 +35,7 @@
             Trying to add a module? See the
             <a
               id="docs-link"
-              href="https://docs.nest.land/eggs/installation.html"
+              href="https://docs.nest.land/eggs/installation"
```

Comment:
```suggestion
              href="https://docs.nest.land/eggs"
```

let's change this to intro page